runtime/parachains/src/configuration.rs

```diff
@@ -613,7 +663,7 @@ pub mod pallet {
 		}
 
 		/// Sets the soft limit for the phase of dispatching dispatchable upward messages.
-		#[pallet::weight((1_000, DispatchClass::Operational))]
+		#[pallet::weight(T::WeightInfo::set_ump_service_total_weight())]
```

Comment:
practically speaking, we could probably make one benchmark for all extrinsics which are simply `new: u32`, which will simplify maintenance a lot.